code/datums/controllers/action_controls.dm

```diff
@@ -1009,31 +1007,47 @@ var/datum/action_controller/actions
 			if(item == source.equipped() && !I)
 				if(target.can_equip(item, slot))
 					logTheThing(LOG_COMBAT, source, "successfully puts \an [item] on [constructTarget(target,"combat")] at at [log_loc(target)].")
-					for(var/mob/O in AIviewers(owner))
-						O.show_message("<span class='alert'><B>[source] puts [item] on [target]!</B></span>", 1)
+					source.visible_message("<span class='alert'><B>[source] puts [item] on [target]!</B></span>", TRUE)
 					source.u_equip(item)
 					if(QDELETED(item))
 						return
 					target.force_equip(item, slot)
 					target.update_inv()
 		else if (I) //Wire: Fix for Cannot execute null.handle other remove().
+			// Picking a clowns pockets is likely to give you a stupid item instead
+			if (target.job == "Clown" && (slot == SLOT_L_STORE) || (slot == SLOT_R_STORE))
+				if (!ON_COOLDOWN(target, "prank_pockets", 30 SECONDS) && prob(50))
```

Comment:
Note that this order of the conditions triggers the cooldown even when the prob fails. Probably what you intended but making sure.

Reply:
Reordered as on second though it's more interesting not to have it get blocked by a successful strip.